api/integrations/asana/zulip_asana_mirror

```diff
@@ -67,20 +69,22 @@ client = zulip.Client(email=config.ZULIP_USER, api_key=config.ZULIP_API_KEY,
                       site=config.ZULIP_SITE, client="ZulipAsana/" + VERSION)
 
 def fetch_from_asana(path):
+    # type: (str) -> Optional[Dict[str, Any]]
     """
     Request a resource through the Asana API, authenticating using
     HTTP basic auth.
     """
-    auth = base64.encodestring('%s:' % (config.ASANA_API_KEY,))
+    auth = base64.encodestring(b'%s:' % (config.ASANA_API_KEY,))
     headers = {"Authorization": "Basic %s" % auth}
 
     url = "https://app.asana.com/api/1.0" + path
-    request = urllib.request.Request(url, None, headers)
-    result = urllib.request.urlopen(request)
+    request = request.Request(url, None, headers)
+    result = request.urlopen(request)
```

Comment:
(This is also going to fail lint because `request` is overwritten with a different type)